lib/api/client/index.js

```diff
@@ -1,3 +1,150 @@
 'use strict'
 
-module.exports = require('../../../src/swagger/client')
+const _             = require('lodash')
+const uuidv4        = require('uuid/v4')
+const jr            = require('../../jr')
+const mock          = require('./mock')
+const EventEmitter  = require('events')
+const SwaggerClient = require('swagger-client')
+
+const KeepAliveAgent   = require('agentkeepalive')
+const agentSettings = {
+  maxSockets:                 100,
+  maxFreeSockets:             10,
+  timeout:                    600000,
+  freeSocketKeepAliveTimeout: 300000
+}
+
+const IS_TEST_ENVIRONMENT = [ 'test', 'gitlab' ].indexOf(process.env.NODE_ENV) > -1
+
+class Client extends EventEmitter {
+
+  initialize(config) {
+    this.config         = config
+    this.services       = this.config.services
+    this.rootPath       = process.cwd()
+    this.swaggerClients = {}
+    this.agent          = new KeepAliveAgent(agentSettings)
+
+    return this._buildClientOperations()
+  }
+
+  async _buildClientOperations() {
+    if (this.services) {
+      for (let s in this.services) {
+        const config = this.services[s]
+        const spec   = require(`${this.rootPath}/${config.spec}`)
+        spec.host    = config.host
+
+        await this._buildOperations({ spec, config })
+      }
+    }
+  }
+
+  async _buildOperations({ spec, config }) {
+    const serviceName = config.name
```

Comment:
+        const spec   = require(`${this.rootPath}/${config.spec}`)
+        spec.host    = config.host